SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -117,9 +124,13 @@ - (void)sdl_applicationWillEnterForeground:(NSNotification *)notification {
 #pragma mark - Stream Lifecycle
 
 - (void)connect {
+    [self sdl_connect:nil];
+}
+
+- (void)sdl_connect:(EAAccessory *)accessory {
```

Comment:
I added some comments to sdl_connectAccessory describing how we handle non-nil and nil accessory parameters.